src/retrace/retrace.py

```diff
@@ -2300,7 +2300,7 @@ def get_bugzillano(self):
         if result is None:
             return None
 
-        return filter(None, set(n.strip() for n in result.split("\n")))
+        return [_f for _f in set(n.strip() for n in result.split("\n")) if _f]
```

Comment:
The `get_bugzillano` loads bugzilla numbers from file.

Can you change the `_f` to `bznum` or something similar?

Reply:
@mkutlak Thanks for the review. I update the patch.